### PR TITLE
add connection_pool optional username field acc tests

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

File: aiven/resource_connection_pool_test.go [hunk 1]
```diff
@@ -83,10 +84,64 @@ func TestAccAivenConnectionPool_basic(t *testing.T) {
...
 	})
 }
 
+func testAccConnectionPoolNoUserResource(name string) string {
+	return fmt.Sprintf(`
+		data "aiven_project" "foo" {
+			project = "%s"
+		}
+
+		resource "aiven_service" "bar" {
```
Comment (mhoffm-aiven) [inline aiven/resource_connection_pool_test.go:108]: we could use aiven_pg here i think

**Review by mhoffm-aiven — APPROVED** [review]
lgtm, i only have a cosmetic suggestion

File: aiven/resource_connection_pool_test.go (hunk 1 above)
Comment (ivan-savciuc) [reply aiven/resource_connection_pool_test.go]: fixes

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**